File: iface-sampleapplication/src/main/java/com/izettle/localconfiguration/iface/sampleapplication/MainActivity.java
package com.izettle.localconfiguration.iface.sampleapplication;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.izettle.localconfiguration.iface.sampleapplication.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding activityMainBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        activityMainBinding = DataBindingUtil.setContentView(this, R.layout.activity_main);

        setSupportActionBar(activityMainBinding.toolbar);
    }

    @Override
    protected void onResume() {
        super.onResume();

        ConfigurationIface configurationIface = new ConfigurationBuilder().create(this, ConfigurationIface.class);

        configurationIface.welcomeTitle();

    }
}

File: wrench-sample/src/main/java/com/example/wrench/MainActivity.kt
package com.example.wrench

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import androidx.navigation.findNavController
import androidx.navigation.ui.NavigationUI

class MainActivity : AppCompatActivity() {

    lateinit var activityMainBinding: com.example.wrench.databinding.ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        activityMainBinding = DataBindingUtil.setContentView(this, R.layout.activity_main)

        val navController = findNavController(R.id.nav_host_fragment)

        NavigationUI.setupWithNavController(activityMainBinding.bottomNav, navController)
    }

    override fun onSupportNavigateUp(): Boolean {
        return findNavController(R.id.nav_host_fragment).navigateUp()
    }
}
